.github/workflows/automation.yml

```
@@ -0,0 +1,197 @@
+        echo "- \`bun run test:safe\` - Fix Vitest then run tests" >> automation-report.md
+        echo "- \`bun run quality:auto\` - Full quality automation" >> automation-report.md
+        
+    - name: Upload automation report
+      uses: actions/upload-artifact@v3
+      with:
+        name: automation-report
+        path: automation-report.md
+        
+    - name: Comment PR with results
+      if: github.event_name == 'pull_request'
+      uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)